packages/web-react/src/components/Flex/__tests__/Flex.test.tsx

```diff
@@ -0,0 +1,123 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import Flex from '../Flex';
+
+describe('Flex', () => {
+  const text = 'Hello world';
+  const testId = 'flex-test-id';
+
+  classNamePrefixProviderTest(Flex, 'Flex');
+
+  stylePropsTest(Flex);
+
+  restPropsTest(Flex, 'div');
+
+  it('should render text children', () => {
+    render(<Flex data-testid={testId}>{text}</Flex>);
+
+    expect(screen.getByText(text)).toBeInTheDocument();
+    expect(screen.getByTestId(testId)).toHaveClass(
+      'Flex Flex--noWrap Flex--row Flex--alignmentXStretch Flex--alignmentYStretch',
+    );
+  });
+
+  it('should have direction className', () => {
```

Comment:
```suggestion
  it('should have direction class name', () => {
```

It is not the name of the prop. We are testing the `class` attribute.